app/views/organizations/tasks/_overview.html.erb

```diff
@@ -98,7 +98,13 @@
               </div>
             </div>
           </li>
-
+          <li class="list-group-item px-0 pb-0 d-flex justify-content-between">
+            <span class="fw-bold">
+              <i class="fe fe-book me-2"></i>
+              Published
+            </span>
+            <%= tag.span "#{@pet.is_published}".titleize, class: "fw-semibold text-dark"%>
+          </li>
```

Comment:
The other `<li>` entries have a cumbersome structure and we can achieve the same result with this